lms/envs/common.py

```diff
@@ -365,7 +365,7 @@
     'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
 
     # WIP -- will be removed in Ticket #TNL-4750.
-    'ENABLE_TIME_ZONE_PREFERENCE': False,
+    'ENABLE_TIME_ZONE_PREFERENCE': True,#FIXME: set this back to False
```

Comment:
I like to make only-for-development changes in a separate commit, that way it's super easy to just drop them when doing a final squash via interactive rebase.